lib/default.nix

```diff
@@ -403,42 +403,101 @@ let
                   gawk
                   coreutils-full
                 ];
+                # This function uses $<zero-width joiner>{ instead of $${ or $\{ to avoid shell interpolation because
+                # nix will print the $$ or \{ literally, making the output more confusing.
+                # The first empty line ensures the text is not indented.
+                warnDeprecated = oldAttr: newAttr: scriptName: lib.warn ''
+
+                  The output .${oldAttr} is deprecated in favor of .${newAttr} and will be removed in
+                  version 2.0.0! To fix this issue, you need to do the following:
+                    - If you're currently calling "$‍{system.build.${oldAttr}}" as a script,
+                      use "$‍{system.build.${newAttr}}/bin/${scriptName}" instead!
+                    - If you added "system.build.${oldAttr}" to your systemPackages, users.<name>.packages
+                      or home.packages, or used it with buildEnv or mkShell, this has never worked
+                      and using "system.build.${newAttr}" will fix it.
+                '';
               in
               lib.mapAttrs throwIfNoDisksDetected {
-                destroyScript = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "disko-destroy" ''
+                diskoDestroy = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-destroy" ''
                   export PATH=${lib.makeBinPath destroyDependencies}:$PATH
                   ${cfg.config._destroy}
                 '';
+                diskoFormat = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-format" ''
+                  export PATH=${lib.makeBinPath (cfg.config._packages pkgs)}:$PATH
+                  ${cfg.config._create}
+                '';
+                diskoMount = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-mount" ''
+                  export PATH=${lib.makeBinPath (cfg.config._packages pkgs)}:$PATH
+                  ${cfg.config._mount}
+                '';
+                diskoDisko = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-disko" ''
```

Comment:
```suggestion
                disko = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-disko" ''
```

Reply:
This just side-steps the issue that `disko` is a bad name for this mode.

I agree that we need a better name and that `diskoDisko` is awkward (though also funny), but just going back to `disko` is not helpful. The only thing that should be called `disko` without a suffix is the CLI itself.

Reply:
I don't think that's a good idea. We can call the output `destroyFormatMount`, but just calling it `disko` makes it too confusing what it's supposed to be, and goes against your other idea of making the outputs a generic interface that has nothing to do with disko directly.

The only output I would expect to be called `disko` is the CLI.